### ISSUE TITLE
Palette tremolo property definition retention

### ISSUE THREAD

**Issue by zacjansheski** [musescore/MuseScore#17243]
### Issue type

UI bug

### Bug description

Tremolos do not retain "play" property definition when adding to a palette.

This does work for trills and I would expect it to work for tremolos as well.

### Steps to reproduce

1. Place a tremolo on a note
2. Turn off "play" in properties
3. Cmnd+Shift+drag tremolo element to palette
4. Add new tremolo to a note from the palette
5. "Play" will be turned on again

### Screenshots/Screen recordings

https://user-images.githubusercontent.com/69917893/232092255-7580991d-bd3f-428f-8346-a7e2ac84bf0a.mp4

### MuseScore Version

4.0.2-230651546

### Regression

No.

### Operating system

macOS11.7

**Comment by oktophonie** [comment]
Related (or at least similar): https://github.com/musescore/MuseScore/issues/17202

**Comment by bkunda** [comment]
fwiw articulation marks like staccato and accent do retain the "play" property when dragged in this way onto a custom palette.

But more importantly, I suspect an aspect of the underlying cause here is that tremolo marks don't seem to respond to the **play** property at all (Indeed, there was no **play** property in MS3.6 for tremolo marks).

After fiddling with this in MS4, it seems that toggling **play** with a tremolo selected actually toggles the **play** property for the note, _not_ the tremolo mark:

https://user-images.githubusercontent.com/86290556/232838348-34dae960-73c6-49d0-962a-53581fc74bbd.mov

So this may end up being a feature request: to support the **play** toggle for tremolo marks.

...unless @zacjansheski there's something I'm missing here?

**Comment by zacjansheski** [comment]
@bkunda you are correct on all accounts.  Does this need to be moved to feature requests?
